### ISSUE TITLE
Multiple style presets

### ISSUE THREAD

**Issue by tmcw** [placemark/figma-plugin#23]

**Comment by akre54** [comment]
Thoughts about allowing a custom map style?

**Comment by tmcw** [comment]
Yeah, definitely considering it. The main question on my mind is whether this UI will have to handle maybe defining a custom format for the style system, or if it should be a UI feature, letting people select colors with e.g. the HTML native color picker, or whether there's a more native Figma way to do this, like color variables.

**Comment by akre54** [comment]
Hmm. Good q. For my purposes a stylejson would suffice, but I could definitely see the use case for wanting to set up a mapping to your design system
